capture.go

```diff
@@ -0,0 +1,84 @@
+package main
+
+import (
+	"time"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/google/gopacket"
+	"github.com/google/gopacket/layers"
+	"github.com/google/gopacket/pcap"
+)
+
+type tcpCapture struct {
+	device   string
+	closed   chan bool
+	packages chan<- *NetPackage
+	handle   *pcap.Handle
+}
+
+func StartTcpCapture(device string, target chan<- *NetPackage) (*tcpCapture, error) {
```

Comment:
actually it's more 'Run' then i think if we use an interface 'PackageCapturer' or something